src/components/TaskCategoryTabSection.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {useRecoilValue} from "recoil";
+import {themeStyleState} from "../state/theme";
+
+export class TaskCategoryTab {
```

Comment:
TaskCategoryTab looks unnecessary. 
The title property is a duplicate of the `TaskSection`'s props.title